### DISCUSSION TITLE
通过何种手段可以监测Doris表数据变化

### DISCUSSION THREAD

**Discussion by wangyjx** [apache/doris#25670] [RFC]
Doris可以使用Flink CDC通过其他数据库的binlog获取其他数据库的更新，那么Doris自己表的数据变化有没有方式可以监测变化并通知下游呢？

比如Doris表数据更新了，我想通知下游BI大屏数据更新，不想用轮询的方式，有没有什么方法。

**Comment by LemonLiTree** [comment]
Not currently. 
You may have to wait until the binlog is available, and then use the binlog to determine whether the data has been updated.

**Reply by ediconss** [reply]
Can use it now？

**Reply by yagagagaga** [reply]
It may can help you if you `show partitions from <table_name>` and sort by `VisibleVersionTime`.

**Reply by ediconss** [reply]
Is binlog still unavailable？

**Reply by yagagagaga** [reply]
It still an experimental feature and will be released when it ready.
